docs/app/WiFiGeolocation.rst

```diff
@@ -0,0 +1,25 @@
+.. _app-wifigeolocation:
+
+WiFi geolocation
+################
+
+.. contents::
+   :local:
+   :depth: 2
+
+If wi-fi site survey geolocation (ground fix service) is implemented, the devices can be shown on the map based on the location of their nearest wi-fi access points.
+
+
+Third-party location APIs
+*************************
+
+nRF Asset Tracker implements the following third-party location APIs:
+
+ - nRF Cloud Location Services
+
+These APIs enable calculation of the rough location of a device as soon as it sends the nearest wi-fi access points information to the cloud.
+
+Follow the configuration guide in the respective implementation to enable nRF Cloud's group fix API:
```

Comment:
Should this be ground fix, not group fix?

Reply:
I believe Ground Fix API is the right thing, note also the capitalization because that is how it is in the API docs.